kubernetes-extension/src/main/java/org/ballerinax/kubernetes/processors/DeploymentAnnotationProcessor.java

```diff
@@ -268,6 +276,50 @@ private ProbeModel parseProbeConfiguration(BLangExpression probeValue) throws Ku
         return null;
     }
 
+    /**
+     * Get Deployment strategy.
+     *
+     * @param keyValue Value of strategy field of deployment annotation.
+     * @return DeploymentStrategy..
+     */
+    private DeploymentStrategy getStrategy(BLangRecordLiteral.BLangRecordKeyValueField keyValue)
+            throws KubernetesPluginException {
+        DeploymentStrategy strategy = new DeploymentStrategy();
+
+        if (keyValue.getValue().type.getKind() == TypeKind.STRING) {
```

Comment:
If the value comes as `kubernetes: STRATEGY_RECREATE` would this work ?

Reply:
Yes.